<?xml version="1.0"?>
<launch>

	<arg name="des_x" default= "-4"/>
	<arg name="des_y" default= "-3"/>
	<arg name="state" default= "4"/>
        <arg name="change_state" default= "0"/>


	<param name="des_pos_x" value="$(arg des_x)" />
	<param name="des_pos_y" value="$(arg des_y)" />
	<param name="state" value="$(arg state)" />
        <param name="change_state" value="$(arg change_state)" />



<node pkg="final_assignment" type="wall_follower_service_m.py" name="wall_follow_service"/> 
<node pkg="final_assignment" type="random_position.py" name="random_position" /> 
<node pkg="final_assignment" type="user_interface.py" name="user_interface" output="screen" />
<node pkg="final_assignment" type="main_node.py" name="main_node" output="screen" />    

</launch>
